runtime/tests/jvmtitests/agent/version.c

```diff
@@ -23,57 +23,13 @@
 
 #include "jvmti_test.h"
 
-const static char *versionNames[] =
-{
-	"JVMTI 1.0",
-	"JVMTI 1.1",
-	"JVMTI 1.2",
-	"JVMTI 9.0",
-	"JVMTI 11",
-	"JVMTI 15",
-	"unknown"
-};
-
-
 jboolean
 ensureVersion(agentEnv * agent_env, jint version)
 {
-
 	if ((agent_env->jvmtiVersion & ~JVMTI_VERSION_MASK_MICRO) == version) {
-		const char *versionName = getVersionName(agent_env, version);
-
-		error(agent_env, JVMTI_ERROR_UNSUPPORTED_VERSION, "Test requires %s",
-				versionName ? versionName : "Unknown Version");
-
+		error(agent_env, JVMTI_ERROR_UNSUPPORTED_VERSION, "Test requires %x", version);
```

Comment:
`%p` is not appropriate for a value of type `jint`.
Also, why is it an error if the version matches? Should it be `!=` instead of `==`?

Reply:
`GetVersionNumber()` returns either `JVMTI_VERSION` or `JVMTI_1_2_3_SPEC_VERSION`, `ensureVersion(agent_env, JVMTI_VERSION_1_1)` checks current `agent_env->jvmtiVersion` is not `JVMTI_VERSION_1_1`.

Reply:
Then I suggest the function is badly named; it ought to be `ensureNotVersion()` and the message when it fails is wrong.
Can you open an issue to clean that up, please?

Reply:
Sure, created https://github.com/eclipse-openj9/openj9/issues/13235